src/js/components/PageHeader/PageHeader.js

```diff
@@ -0,0 +1,82 @@
+import React, { forwardRef, useContext } from 'react';
+import { ThemeContext } from 'styled-components';
+
+import { Box } from '../Box';
+import { Header } from '../Header';
+import { Heading } from '../Heading';
+import { Grid } from '../Grid';
+import { Paragraph } from '../Paragraph';
+import { ResponsiveContext } from '../../contexts/ResponsiveContext';
+
+// QUESTIONS:
+// 1. do we want the heading / paragraph to fill or let them wrap?
+// 2. should the rest go on the header or the grid?
```

Comment:
To me it feels more intuitive to have the rest on the Header. But maybe we should allow for passing additional grid props through `gridProps` similar to how on DropButton you can pass any additional drop props through `dropProps`

Reply:
That could play nicely, I like that suggestion.

Reply:
@jcfilben's comment was the direction I had been thinking as well.

Reply:
Done.